crates/cxx-qt-gen/src/generator/structuring/mod.rs

```diff
@@ -29,15 +31,30 @@ impl<'a> Structures<'a> {
     /// Create a new `Structures` object from the given `ParsedCxxQtData`
     /// Returns an error, if any references could not be resolved.
     pub fn new(cxxqtdata: &'a ParsedCxxQtData) -> Result<Self> {
+        let methods_found: Vec<_> = cxxqtdata
+            .methods
+            .iter()
+            .map(|method| (method.name.clone(), method.qobject_ident.clone()))
+            .collect();
+        for method_pair in methods_found {
```

Comment:
i know it's just tracing but remember you can destructure :-)

`for (key, value) in methods_found {`